devops/actions/aws-ec2/aws-ec2.js

```diff
@@ -35,7 +35,13 @@ let reg_token;
 
 // starts AWS EC2 instance that will spawn Github runner for a given label
 async function start(param_type, param_label, param_ami, param_spot, param_disk, param_timebomb, param_onejob) {
-  const ec2 = new AWS.EC2();
+  const ec2 = new EC2({
```

Comment:
Please move this code for ec2 object creation outside this function (ideally it should be only called only once before start ort stop function call) since this function can be called in the loop and creating several copies of this EC2 object is not the best practice I guess.